packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -199,7 +199,7 @@ const DomainStep: React.FunctionComponent< DomainStepProps > = ( {
 						<p className="focused-launch-summary__side-commentary-title">
 							{ createInterpolateElement(
 								__(
-									'<strong>Unique domains</strong> help build brand recongnition and trust',
+									'<strong>Unique domains</strong> help build brand recognition and trust',
```

Comment:
Thank you for fixing my typo!

Reply:
Highly recommend the "Code Spell Checker" extension. Saves you from bugs too!